client/src/components/common/applicant-info-1.vue

```diff
@@ -568,10 +597,24 @@ export default class ApplicantInfo1 extends Vue {
       (this.$refs.step1 as any).validate()
     }
   }
+
+  @Watch('isValid')
```

Comment:
I watch `isValid` to determine when the buttons become "true".

Note that this component does NOT get destroyed when the user goes to the next/prev page, which is why this code is needed.